website/static/website/style.css

```diff
@@ -7062,3 +7062,298 @@ a, a:visited, a:hover {
     transition: 0.3s; }
 
 /*# sourceMappingURL=style.css.map */
+=======
```

Comment:
You forgot to take out a merge conflict here. Just delete this file. We can regenerate it after we merge this.